lib/v2/screens/profile/support/support_screen.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/i18n/support.i18n.dart';
+
+class SupportScreen extends StatelessWidget {
+  const SupportScreen({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text('Support'.i18n)),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          children: [
+            InkWell(
+              borderRadius: BorderRadius.circular(defaultCardBorderRadius),
+              onTap: () {},
```

Comment:
please ask Jason what should happen when the user taps this view.

Reply:
@JGDoh This screen sends the mobile chrome app?

Reply:
@gguijarro-c-chwy on press action updated